### ISSUE TITLE
Cyber Shadow [0100C1F0141AA000] writes huge amount of debug stuff into the logfile making the game running slow

### ISSUE THREAD

**Issue by F1ReB4LL** [Ryujinx/Ryujinx#3467]
### What's the issue you encountered?

Ryujinx writes a huge chunk of text into the logfile every millisecond, as a result, the logfile grows by around 15 megabytes every minute causing game stuttering. Doesn't happen with other games, looks like some forgotten debug logging?

> 00:00:12.383 |G| HLE.OsThread.16 ServiceLm Log: Guest Log:
Log level: Info
Time: 12s
Line: 0
Filename: ''
Function: ''
Module: ''
Thread: 'MainThread'
ProgramName: 'Chowdren'
Message: ''

> 00:00:12.383 |G| HLE.OsThread.16 ServiceLm Log: Guest Log:
Log level: Info
Time: 12s
Line: 0
Filename: ''
Function: ''
Module: ''
Thread: 'MainThread'
ProgramName: 'Chowdren'
Message: 'get uniform:'

> 00:00:12.383 |G| HLE.OsThread.16 ServiceLm Log: Guest Log:
Log level: Info
Time: 12s
Line: 0
Filename: ''
Function: ''
Module: ''
Thread: 'MainThread'
ProgramName: 'Chowdren'
Message: 'WindowsWidth'

and so on
### How can the issue be reproduced?

Just run the game, starts instantly.

### Log file

[Ryujinx_1.1.178_2022-07-14_05-58-52.log](https://github.com/Ryujinx/Ryujinx/files/9108070/Ryujinx_1.1.178_2022-07-14_05-58-52.log)

### Environment?

- Ryujinx version: 1.1.178</br>
- Game version: 1.0.4</br>
- System Specs:
  - OS: *Windows 10*
  - CPU: *i5-11400H*
  - GPU: *NVIDIA RTX 3050*
  - RAM: *24GB*
- Applied Mods : No

**Comment by gdkchan** [comment]
Guest logs are produced by the game. If you don't want them being added to the log, you can disable guest logs in the settings, Logging tab.

**Comment by F1ReB4LL** [comment]
So it's a bad game coding? If to disable the guest logs, it still spams with TONS of "HLE.OsThread.16 ServiceHid Start" and "HLE.OsThread.9 ServiceHid SetSupportedNpadStyleSet" messages, so you need to disable all the logging to play it normally.

**Comment by F1ReB4LL** [comment]
Baba Is You - 01002CD00A51C000 and Breakout: Recharged - 010022C016DC8000 have the similar issues, I think (lots of logged events + both work slow).

**Comment by gdkchan** [comment]
I believe its just something wrong with your controller settings since the games are constantly calling the controller applet. I would recommend deleting the Config.json on the Ryujinx AppData folder file and re-configuring it.

**Comment by F1ReB4LL** [comment]
Yeah, probably. When you run Cyber Shadow with only P1 configured and set to Joycon Pair, you're getting a nag screen telling you that you can't play with Joycon Pair ("Application requires exactly 1 player(s) with..."). And if you change the controller type to the "allowed" one, the game works fast and only Stub messages appear in both console and log outputs. BUT if you enable P2 as Joycon Right (for example), you can set Joycon Pair for P1 and you won't get any nag screens, but you will get lots of Guest log messages and game slowdowns/hiccups/microfreezes. That nag screen probably needs additional checks to not to allow such a combination :) Will recheck other 2 games later.
